python/build_definitions/bison.py

```diff
@@ -28,9 +28,10 @@ def __init__(self) -> None:
     def get_additional_compiler_flags(self, builder: BuilderInterface) -> List[str]:
         llvm_major_version = builder.compiler_choice.get_llvm_major_version()
         flags = []
-        if (is_linux() and
-                llvm_major_version is not None
-                and llvm_major_version >= 16):
+        linux_llvm16_or_later = (
+            is_linux() and llvm_major_version is not None and llvm_major_version >= 16)
```

Comment:
Why do you need to change this if you're already setting `Wno-error=incompatible-function-pointer-types` in gettext.py?

Reply:
We set it per build. Its not persisted across different build definitions